patches/common/chromium/chore_add_debounce_on_the_updatewebcontentsvisibility_method_to.patch

```diff
@@ -0,0 +1,65 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Wed, 5 Jun 2019 15:11:00 -0700
+Subject: chore: add debounce on the updateWebContentsVisibility method to
+ ensure quick changes in occlusion do not result in flickering
```

Comment:
Could you potentially copy the PR body into the extended commit message for posterity?